app/Main.hs

```diff
@@ -71,10 +74,16 @@ main :: IO ()
 main = do
   setSGR [SetColor Foreground Vivid Blue]
   putStrLn "Exporting Plutarch scripts..."
-
   setSGR [Reset]
-  writePlutusScript "Smart Handle Router" "./compiled/smartHandle.plutus" psmartHandleValidatorW
-  putStrLn "Exported smart contract handle router"
+
+  writePlutusScript "Smart Handle" "./compiled/smartHandleSimple.plutus" psmartHandleValidatorW
```

Comment:
let's make all  *.plutus  ->  to *.json, it's easier when importing the file in the off-chain code